pages/common/ect.md

```diff
@@ -1,16 +1,25 @@
 # ect
 
-> Efficient Compression Tool (or ECT) is a C++ file optimizer. It supports PNG, JPEG, GZIP and ZIP files.
+> Efficient Compression Tool.
+> File optimizer written in C++. It supports `.png`, `.jpg`, `.gzip` and `.zip` files.
 > More information: <https://github.com/fhanau/Efficient-Compression-Tool>.
 
 - Compress a file:
 
-`ect {{filename.png}}`
+`ect {{path/to/file.png}}`
 
-- Compress a file with the highest compression level and multithreading:
+- Compress a file with specified compression level and multithreading (1=Fastest (Worst), 9=Slowest (Best), default is 3):
 
-`ect -9 --mt-deflate {{filename.png}}`
+`ect -{{1|2|3|4|5|6|7|8|9}} --mt-deflate {{path/to/file.zip}}`
 
-- Compress all the files in a directory recursively, keeping the original modification time:
+- Compress all files in a directory recursively:
 
-`ect -keep -recurse {{directory}}`
+`ect -recurse {{path/to/directory}}`
+
+- Keep the original modification time:
+
+`ect -keep {{path/to/file.png}}`
+
+- Strip metadata:
+
+`ect -strip {{path/to/file.png}}`
```

Comment:
Does this compress the file too?

Reply:
Yes, I see what you're getting at.  I actually did do here what you did above, but...